src/checkstyle/checkstyle.xml

```diff
@@ -153,6 +153,9 @@
         <!-- See http://checkstyle.sf.net/config_blocks.html -->
         <module name="AvoidNestedBlocks"/>
         <module name="EmptyBlock"/>
+        <module name="EmptyCatchBlock">
+            <property name="exceptionVariableName" value="expected"/>
+        </module>
```

Comment:
This is what prompted the `checkstyle`/`maven-checkstyle-plugin` upgrade — I think it's relatively new. Definitely nice than wrapping the whole thing in `ON`/`OFF` comments.